package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yang-sprotty",
-  "version": "0.1.9",
+  "version": "0.2.0",
```

Comment:
The version should be lifted up when we do the next release, or what do you think, @JanKoehnlein?

Reply:
Better now.